### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by benmliang**
Some places stumped me.. 

I will take a look later, gotta go to gym first :strong:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ const state = {
     innerProp: 'test',
     },
   }
+  
+// we want to transform it to:
+{  
+  one: '1',
+  innerProp: 'test',
+}
```
Comment (benmliang) [inline README.md:24]: I want to know what you want to achieve here :dancer:

File: README.md [hunk 2]
```diff
@@ -49,25 +55,33 @@ const myTransformer = createTransformer(
 const result = myTransformer(initialObj, state);
 
 //=> {
-  first: '1',
+  one: '1',
```
Comment (benmliang) [inline README.md:58]: I think the property name should be `one`

File: README.md [hunk 3]
```diff
@@ -49,25 +55,33 @@ const myTransformer = createTransformer(
...
-const selectorToCallWithState(state); // gets same result
+const selectorToCallWithState(state); // gets same result 
+
+// Ben: I don't understand this example really well.
+* I don't know what is `initialObj`
+* you didn't assign a value to `myTransformer` 
+* you mean selectorToCallWithState(state) === myTransformer(initialObject)
+* Now I know myTransformer, initialObj is from the previous example
+* maybe make each example more independent or just put into one example or ___?
 ```
```
Comment (benmliang) [inline README.md:85]: This is my first impression. I know, it is hard to find a good example

File: README.md [hunk 4]
```diff
@@ -49,25 +55,33 @@ const myTransformer = createTransformer(
 const result = myTransformer(initialObj, state);
 
 //=> {
```
Comment (benmliang) [inline README.md:57]: the first part of bracket gets comment out

**Comment by NicholasGWK** [comment]
Thanks Ben, made some updates! Let me know what you think

File: README.md (hunk 2 above)
Comment (BrendanFDMoore) [reply README.md:58]: Yep, agreed. :100:

**Pull request closed without merging**